rootfs/rootfs/usr/local/etc/init.d/docker

```diff
@@ -20,6 +20,9 @@ test -f '/var/lib/boot2docker/profile' && . '/var/lib/boot2docker/profile'
 : ${SERVERKEY:="${CERTDIR}serverkey.pem"}
 : ${CERT:="${CERTDIR}cert.pem"}
 : ${KEY:="${CERTDIR}key.pem"}
+: ${ORG:=Boot2Docker}
+: ${SERVERORG:="${ORG}"}
+: ${CAORG:="${ORG}CA"} # Append 'CA'; see <http://tinyurl.com/pqgnb2a>
```

Comment:
Can you please replace this tinyurl with the full `http://rt.openssl.org/Ticket/History.html?use
r=guest&pass=guest&id=3979` ?